### PR TITLE
Add v2 blueprints

### PR DESCRIPTION
**Pull request by pavannd1**
## Pull request type

Please check the type of change your PR introduces:
- [ ] :construction: Work in Progress
- [ ] :rainbow: Refactoring (no functional changes, no api changes)
- [ ] :hamster: Trivial/Minor
- [ ] :bug: Bugfix
- [ ] :sunflower: Feature
- [ ] :world_map: Documentation
- [ ] :robot: Test

## Issues

- #XXX

## Test Plan

- [ ] :muscle: Manual
- [ ] :zap: Unit test
- [ ] :green_heart: E2E

### CODE REVIEW

**Review by PrasadG193 — APPROVED** [review]

File: examples/stable/elasticsearch/blueprint-v2/elasticsearch-blueprint.yaml [hunk 1]
```diff
@@ -0,0 +1,75 @@
+apiVersion: cr.kanister.io/v1alpha1
+kind: Blueprint
+metadata:
+  name: elasticsearch-blueprint
+actions:
+  backup:
+    type: StatefulSet
```
Comment (viveksinghggits) [inline examples/stable/elasticsearch/blueprint-v2/elasticsearch-blueprint.yaml:7]: should we remove these `type` fields now. I don't think we are using them anywhere.

File: examples/stable/foundationdb/blueprint-v2/foundationdb-blueprint.yaml [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+metadata:
+  name: foundationdb-blueprint
+actions:
+  backup:
+    kind: CustomResource
+    outputArtifacts:
+      fdbBackup:
+        keyValue:
+          backupPath: "{{ .Phases.TakeBackup.Output.backupPath }}"
+          backupID: "{{ .Phases.TakeBackup.Output.backupID }}"
```
Comment (viveksinghggits) [inline examples/stable/foundationdb/blueprint-v2/foundationdb-blueprint.yaml:12]: I am assuming we are setting this in code, I dont see this being set in the blueprint.

**Review by viveksinghggits — COMMENTED** [review]
Looks good to me.

File: examples/stable/maria/blueprint-v2/maria-blueprint.yaml [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+        - |
+          backup_path="/maria-backups/{{ .StatefulSet.Namespace }}/{{ .StatefulSet.Name }}/{{ toDate "2006-01-02T15:04:05.999999999Z07:00" .Time  | date "2006-01-02T15-04-05" }}/dump.sql"
+          root_password="{{ index .Phases.dumpToStore.Secrets.mariaSecret.Data "mariadb-root-password" | toString }}"
+          exclude_db="('information_schema', 'mysql', 'performance_schema')"
+          dump_databases=""
+          for db in `mysql -u root  -h {{ .StatefulSet.Name }} --password=${root_password} -ANe "use information_schema; select schema_name from schemata where schema_name not in ${exclude_db}"`
+          do
+                  dump_databases="${dump_databases} ${db}"
+          done
+          mysqldump -u root --password=${root_password} -h {{ .StatefulSet.Name }} --column-statistics=0 --single-transaction --databases ${dump_databases} | gzip - | kando location push --profile '{{ toJson .Profile }}' --path ${backup_path} -
```
[the remark this answers is not in the document]
Comment (PrasadG193) [reply examples/stable/maria/blueprint-v2/maria-blueprint.yaml:40]: I've made the change and manually validated, it is working as expected

File: examples/stable/foundationdb/blueprint-v2/foundationdb-blueprint.yaml (hunk 2 above)
Comment (PrasadG193) [reply examples/stable/foundationdb/blueprint-v2/foundationdb-blueprint.yaml:12]: That's right, kando sets this internally when Kopia location profile is used

Comment (PrasadG193) [reply examples/stable/foundationdb/blueprint-v2/foundationdb-blueprint.yaml:12]: ref: https://github.com/kanisterio/kanister/blob/master/pkg/kando/location_push.go#L91

File: examples/stable/elasticsearch/blueprint-v2/elasticsearch-blueprint.yaml (hunk 1 above)
Comment (PrasadG193) [reply examples/stable/elasticsearch/blueprint-v2/elasticsearch-blueprint.yaml:7]: We will address this in separate PR as we may want to remove it from older BPs as well

**Pull request merged**